photon-client/src/main/scala/com/linkedin/photon/ml/hyperparameter/ShrinkSearchRange.scala

```diff
@@ -0,0 +1,128 @@
+package com.linkedin.photon.ml.hyperparameter
+import breeze.linalg.{DenseMatrix, DenseVector}
+import com.linkedin.photon.ml.hyperparameter.estimators.GaussianProcessEstimator
+import com.linkedin.photon.ml.hyperparameter.estimators.kernels.Matern52
+import org.apache.commons.math3.random.SobolSequenceGenerator
+
+import scala.math.{floor, min, max}
```

Comment:
Imports should be grouped as below, separated by newline:

1. Langage runtime packages
2. 3rd-party packages
3. Current project packages

E.g., for the above, it would be:

```scala
import scala.math.{floor, min, max}

import breeze.linalg.{DenseMatrix, DenseVector}
import org.apache.commons.math3.random.SobolSequenceGenerator

import com.linkedin.photon.ml.hyperparameter.estimators.GaussianProcessEstimator
import com.linkedin.photon.ml.hyperparameter.estimators.kernels.Matern52
```

Reply:
Just to be a little picky, please sort it by alphabetic order. For example,
`import scala.math.{floor, max, min}`
:)